tnt/src/main/java/org/dcsa/conformance/standards/tnt/party/DynamicScenarioParameters.java

```diff
@@ -0,0 +1,47 @@
+package org.dcsa.conformance.standards.tnt.party;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import lombok.NonNull;
+import lombok.With;
+
+import static org.dcsa.conformance.core.toolkit.JsonToolkit.OBJECT_MAPPER;
+
+@With
+public record DynamicScenarioParameters(
```

Comment:
Hi @preetamnpr, in the commit of yesterday afternoon, with the message "STNG-184 Improve all DSP and related classes (#198)" I've improved the DSP code in our whole code base. Can you please checkout those examples? You can do the same for this TNT DSP impl. Let me know if I can help.

Reply:
Yesterday afternoon, the PR was merged. Please compare this class with any other DSP class. You can reduce this class now, to only a few lines. Please let me know if I can help @preetamnpr

Reply:
Sure. I am yet to do the pagination implementation and I would take the latest changes for DSP.

Reply:
Great, thanks! 👍🏻